src/test/components/Root.test.js

```diff
@@ -132,15 +134,29 @@ describe('app/ProfileViewWhenReady', function() {
     navigateBackToHome();
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  it('renders a compare home when navigating to /compare/, then loads when changing profiles', () => {
+    const { container, dispatch, navigateToCompareHome } = setup();
+
+    navigateToCompareHome();
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
I think this snapshot is a duplicate of the `src/test/components/CompareHome.test.js`. Can this be turned into just an existence check for that component?